packages/twenty-docs/docs/user-guide/basics/notes.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Notes
+sidebar_position: 1
+sidebar_custom_props:
+  icon: TbNote
+---
+
+Easily create a note to keep track of important information. 
+
+## Create Notes 
+
+To attach a note to the record, go to the <b>Notes</b> tab of a record page and click on `+ New Note`. You can also format and comment on your notes.   
+
+<div style={{textAlign: 'center'}}>
+    <img src="/img/user-guide/create-new-note.png" alt="Create new note"/>
+</div>
+
+## Format Notes 
+
+You can format your notes right from the editor. Press the `/` key or click on the `+` icon on the editor to see the different block options, such as headings, tables, and lists. You can also attach images to your note. 
+
+Highlight the text to see more formatting options like bold, italics, and alignment options. 
+
+You can also change the background color and text color of each block to highlight important things in your note. To do so, hover over the block you want to format and click on the `⋮⋮` icon besides the `+` icon. Click on <b>Colors</b> to open up all color options for both the text and the background.  
+
```

Comment:
Could we add something add adding an image to a note? (it was possible to add through a link and very soon through upload: https://github.com/twentyhq/twenty/pull/3044)